dom/dom.php

```diff
@@ -9,7 +9,7 @@
  * @param SimpleXMLElement $node <p>
  * The <b>SimpleXMLElement</b> node.
  * </p>
- * @return DOMElement|null|false The <b>DOMElement</b> node added or <b>FALSE</b> if any errors occur.
+ * @return DOMElement|null The added <b>DOMElement</b> node.
```

Comment:
This change seems to be incorrect. `simplexml_load_string('</>');` will return false so seems the correct way would be to change return type-hint instead of phpdoc